sharepoint/sharepoint-ps/sharepoint-online/Get-SPOEnterpriseAppInsightsReport.md

```diff
@@ -0,0 +1,123 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spoenterpriseappinsightsreport
+applicable: SharePoint Online
+title: Get-SPOEnterpriseAppInsightsReport
+schema: 2.0.0
+author: sumikumar
+ms.author: sumikumar
+ms.reviewer:
+manager: hikakar
+---
+
+# Get-SPOEnterpriseAppInsightsReport
+
+## SYNOPSIS
+
+This cmdlet enables the administrator to check status of all active and available reports when no report ID is present and to view or download a report if report ID is present.
+
+## SYNTAX
+
+```powershell
+Get-SPOEnterpriseAppInsightsReport [-ReportId <Guid>] [-Action <ActionType>]
+```
+
+## DESCRIPTION
+
+If this cmdlet is executed without any parameters, it displays the status of all active and completed reports with the following properties:
+
+| Property             | Description                                                 |
+|:---------------------|:------------------------------------------------------------|
+| Id                   | The unique Id of the report.                                |
+| CreatedDateTimeInUtc | The date and time the report creation was triggered in UTC. |
+| Status               | The status of the report.                                   |
+| ReportPeriodInDays   | The report duration in days.                                |
+
+If this cmdlet is executed with `-ReportId` as parameter, the top 100 records of the report from the last N days will be displayed with the following properties:
+
+| Property        | Description                                                                |
+|:----------------|:---------------------------------------------------------------------------|
+| SiteName        | The name of the SharePoint site.                                           |
+| SiteURL         | The URL of the SharePoint site.                                            |
+| SiteSensitivity | The sensitivity label of the SharePoint site.                              |
+| AppID           | The AppID of the 3P application.                                           |
+| AppPermissions  | The permissions granted to the 3P application.                             |
+| RequestVoulme   | The number of times the 3P application accessed the given SharePoint site. |
+
+If this cmdlet is executed with both the parameters, i.e. `-ReportId` and `-Action`, and if the value of `-Action` is set as `View`, it will display the same result as described above. If the value of `-Action` is set to `Download`, it will download the full report in CSV format to the same path from where the command was run.
+  
+> [!NOTE]
+> All reports adhere to any retention timeline as per [Data Access Governance](/sharepoint/sharepointonline/data-access-governance-reports.md).
```

Comment:
>/sharepoint/sharepointonline/data-access-governance-reports.md [](http://example.com/codeflow?start=79&length=62)

this is the right link @Shweta2024 

`/sharepoint/data-access-governance-reports`